Packs/Tenable_io/Integrations/Tenable_io/Tenable_io.py

```diff
@@ -744,6 +744,37 @@ def export_request_with_export_uuid(export_uuid: str, assets_or_vulns: str) -> d
     return res.json()
 
 
+def get_scanner_info():
+    """
+    Returns the scanner list and additional information about each scanner
+    Returns:
+        CommandResults: Command object with relevant data.
+    """
+    full_url = f'{BASE_URL}/scanners'
+    response = requests.get(full_url, headers=HEADERS, verify=USE_SSL)
+    response_data = response.json()
+    response.raise_for_status()
+    if scanners := response_data.get('scanners'):
+        scanner_data = []
```

Comment:
At its current state, if the `scanners` key is missing or empty, the `scanner_data` variable won't exist, which will cause an error when trying to use it for `tableToMarkdown` or `CommandResults`.

```suggestion
    scanner_data = []

    if scanners := response_data.get('scanners'):
```